src/preferences.yaml

```diff
@@ -1,3 +1,5 @@
 user_preferences:
-  wake-up-time-in-minutes: 60
+  wake-up-time-in-minutes: 50
```

Comment:
Was ist die Wakeup time in minutes? Zeit die man zu Aufstehen braucht? Finde den Namen nicht gut genug gewählt.